File: YFDebugPannel/YFDebugPannel/HCEnvSection.m
#import "HCEnvSection.h"
#import "HCCellItem.h"

@implementation HCEnvSection

+ (instancetype)sectionWithTitle:(NSString *)title items:(NSArray<HCCellItem *> *)items {
    HCEnvSection *section = [[HCEnvSection alloc] init];
    section.title = title;
    section.items = items;
    return section;
}

@end

File: YFDebugPannel/YFDebugPannel/HCEnvSection.swift
import Foundation

final class HCEnvSection {
    var title: String
    var items: [HCCellItem]

    init(title: String, items: [HCCellItem]) {
        self.title = title
        self.items = items
    }

    static func section(title: String, items: [HCCellItem]) -> HCEnvSection {
        HCEnvSection(title: title, items: items)
    }
}
